src/icons/imageset.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class IconImageSet extends Component {
+
+  render() {
+    return (
+      <svg id="imageset" data-name="imageset" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44">
```

Comment:
👍  nice and simple way to integrate icons. FWIW if icons become unwieldy to maintain, we could consider a build step to be able to simple embed svg files through webpack/browserify or otherwise.